apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/CreateTableOptions.cs

```diff
@@ -46,15 +46,22 @@ public sealed class CreateTableOptions
         /// <summary>
         /// The expiration duration for each partition in a table.
         /// </summary>
+        [Obsolete("This property will be removed before final release. Please use TimePartition to create a TimePartitioning and set CreateTableOptions.TimePartitioning.")]
         public TimeSpan? TimePartitionExpiration { get; set; }
 
         /// <summary>
         /// The time partition type for the table, if the table should be created with
         /// time-based partitioning.
         /// </summary>
+        [Obsolete("This property will be removed before final release. Please use TimePartition to create a TimePartitioning and set CreateTableOptions.TimePartitioning.")]
         public TimePartitionType? TimePartitionType { get; set; }
 
         /// <summary>
+        /// The time partitioning for the table. For a .NET-friendly 
```

Comment:
Looks like this comment wasn't finished.

Reply:
Fixed, and added something similar to the other options.